crates/primitives/src/bits/macros.rs

```diff
@@ -173,7 +173,7 @@ macro_rules! wrap_fixed_bytes {
             /// Returns the inner bytes array.
             #[inline]
             pub const fn into_array(self) -> [u8; $n] {
-                self.0 .0
+                self.0.0
```

Comment:
This is actually intentional since Rust parses the `0.0` as a float literal and rustfmt will prefer the separated version, but both work

Reply:
Oh ok, had no idea. Thanks for explaining!

Reply:
rustfmt formats `self.0.0` to `self.0 .0` if you try it (top left, tools -> rustfmt):
<https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=fc90885df6ae8d4805352d11061f9a2b>